tools/python/task.yaml

```diff
@@ -18,3 +18,36 @@ tasks:
       - python3 -m pip show poetry &>/dev/null
     cmds:
       - python3 -m pip install poetry
+
+  init_project:
+    desc: Initialize poetry in a new project and install common dev dependencies
+    vars:
+      FLAKE8_CONFIG: |
+        [flake8]
+        ignore = E501
+        max-complexity = 10
+        exclude = .git,__pycache__,.venv,.mypy_cache,venv
+        max-line-length = 120
+        per-file-ignores = __init__.py:F401,__main__.py:E402
+    cmds:
+      - echo "{{.FLAKE8_CONFIG}}" > .flake8
+      - |
+        poetry init \
+          --dev-dependency black \
+          --dev-dependency flake8 \
+          --dev-dependency isort
+
+  init:
```

Comment:
this should be `setup` or something similar?

Reply:
I don't have well formed opinions here. It would be good to have a consistent understanding of what an 'init' or 'setup' does so I'm happy to conform to your expectations.